docs/data-management-on-carbondata.md

```diff
@@ -524,6 +540,9 @@ Users can specify which columns to include and exclude for local dictionary gene
     ```
    ALTER TABLE tablename UNSET TBLPROPERTIES('LOCAL_DICTIONARY_ENABLE','LOCAL_DICTIONARY_THRESHOLD','LOCAL_DICTIONARY_INCLUDE','LOCAL_DICTIONARY_EXCLUDE')
     ```
+    
+   **NOTE:** For old tables, by default, local dictionary is disabled. If user wants local dictionary, user can enable/disable local dictionary for new data on those tables at their discretion. 
```

Comment:
local dictionary is disabled for new tables also.Need to mention it can be enabled for old tables also